assets/javascripts/discourse/components/explorer-schema/enum-info.js

```diff
@@ -0,0 +1,15 @@
+import Component from "@glimmer/component";
+
+export default class ParamInput extends Component {
+  get enuminfo() {
+    const hash = this.args.col.enum;
+    let result = [];
+    for (let key in hash) {
+      if (!hash.hasOwnProperty(key)) {
+        continue;
+      }
+      result.push({ value: key, name: hash[key] });
+    }
+    return result;
```

Comment:
```suggestion
    return Object
      .entries(this.args.col.enum)
      .map(([value, name]) => { value, name });
```

Reply:
This is causing a https://eslint.org/docs/latest/rules/no-sequences linting error. If I understand this correctly, I don't think we will be able to do the computation of `name` within the comma operator here.